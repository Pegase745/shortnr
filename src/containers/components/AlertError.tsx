import * as React from 'react';
import { Message } from 'semantic-ui-react';

interface IProps {
  error: string;
}

const AlertError: React.FC<IProps> = props => (
  <Message error={true} header={props.error} />
);

export default React.memo<IProps>(AlertError);
