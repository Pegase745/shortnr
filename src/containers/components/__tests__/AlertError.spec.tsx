import { shallow } from 'enzyme';
import * as React from 'react';

import AlertError from '../AlertError';

const defaultProps = {
  error: 'A big error',
};

describe('AlertError.tsx', () => {
  it('should render', () => {
    const app = shallow(<AlertError {...defaultProps} />);

    expect(app).toMatchSnapshot();
  });
});
