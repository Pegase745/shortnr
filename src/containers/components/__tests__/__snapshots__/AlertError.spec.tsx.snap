// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertError.tsx should render 1`] = `
<Message
  error={true}
  header="A big error"
/>
`;
